frontend/src/components/dialogs/ProjectJoinDialog.tsx

```diff
@@ -0,0 +1,159 @@
+  },
+  adminAvatar: {
+    height: 20,
+    width: 20,
+    display: "inline-block",
+    textAlign: "center",
+    verticalAlign: "middle",
+  },
+  chip: {
+    backgroundColor: "#c4c2c2",
+    borderRadius: "20px",
+    margin: theme.spacing(0.5),
+  },
+}));
+
+export default function ProjectJoinDialog({
+  open,
+  onClose,
+  user,
+  projectAdmin,
+  handleSendProjectJoinRequest,
+  url
+}) {
+  console.log("url",url);
+  
+  const classes = useStyles();
+  const { locale } = useContext(UserContext);
+  const texts = getTexts({ page: "project", locale: locale });
+  const [description, setDescription] = useState("");
+  const [chips, setChips] = useState([
+    { id: 1, label: texts.already_part_of_this_project },
+    { id: 2, label: texts.get_active_to_project },
+  ]);
+
+  const handleClose = () => {
+    onClose();
+  };
+  console.log("projectAdmin",projectAdmin);
+  
+  const avatarProps = {
+    className: classes.adminAvatar,
+    src: getImageUrl(projectAdmin?.image),
+    alt: projectAdmin?.name,
+  };
+
+  const onDescriptionChange = (event) => {
+    setDescription(event.target.value)
+  };
+
+  const handleSelectAnswer = (chipId) => {
+    const updatedChips = chips.filter((chip) => chip.id !== chipId);
+    setChips(updatedChips);
+    setDescription(
+      (prevValue) => prevValue + `${chips.find((chip) => chip.id === chipId).label}`
+    );
+  };
+
+  return (
+    <GenericDialog
+      maxWidth="sm"
+      open={open}
+      onClose={handleClose}
+      title={texts.thrilled_to_join_the_project + "!"}
+      titleTextClassName={classes.dialogTitle}
+    >
+      <>
+        {!user && (
+          <>
+            <Typography>
+              {texts.please_log_in + " " + texts.to_send_this_project_join_request + "!"}
+            </Typography>
+            <Container className={classes.dialogButtonContainer}>
+              <Button
+                className={classes.dialogButton}
+                variant="contained"
+                color="primary"
+                href={getLocalePrefix(locale) + "/signin?redirect=" + encodeURIComponent(url)}
+              >
+                {texts.log_in}
+              </Button>
+            </Container>
+          </>
+        )}
+        {user && (
+          <>
+            <Typography>
+              {texts.please_share_with + " "}
+              <Link href={"/profiles/" + projectAdmin.url_slug} underline="hover">
+                {projectAdmin?.badges?.length > 0 ? (
+                  <ProfileBadge badge={projectAdmin?.badges[0]} size="medium">
+                    <Avatar {...avatarProps} />
+                  </ProfileBadge>
+                ) : (
+                  <Avatar {...avatarProps} />
+                )}
+                {" [" + projectAdmin.name + "] "}
+              </Link>
+              {texts.project_admin + ", " + texts.what_inspires_you_to_be_part_of_this_team + "!"}
+            </Typography>
+            {chips.map((data) => {
+              return (
```

Comment:
The chips should not add text but instead act like a radio button that you can select.